service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseSubmissionValidatorService.java

```diff
@@ -24,27 +24,25 @@
         Validator validator = Validation.buildDefaultValidatorFactory().getValidator();
         Set<ConstraintViolation<CaseData>> violations = validator.validate(caseData);
 
-        Stream.of("applicant", "children", "orders", "groundsForTheApplication", "hearing",
-            "caseName", "documents")
+        Stream.of(SectionType.APPLICANT, SectionType.CHILDREN, SectionType.ORDERS, SectionType.GROUNDS,
+            SectionType.HEARING, SectionType.DOCUMENTS, SectionType.CASENAME)
```

Comment:
I remember Sam suggesting the filtering by the section predicate, but I don't think it's strictly required, the code is quite streamlined now.

Reply:
I'd agree, @swalker125 thoughts?